test/integration/connect/envoy/helpers.windows.bash

```diff
@@ -839,7 +839,7 @@ function gen_envoy_bootstrap {
     -envoy-version "$ENVOY_VERSION" \
     -http-addr envoy_consul-${DC}_1:8500 \
     -grpc-addr envoy_consul-${DC}_1:8502 \
-    -admin-access-log-path C:\\log \
+    -admin-access-log-path="C:/envoy/envoy.log" \
```

Comment:
Is this tested and working?

Reply:
Yes, it is working. I have tested it manually